graylog2-web-interface/src/views/components/datatable/DataTable.tsx

```diff
@@ -110,11 +137,19 @@ const DataTable = ({
   condensed,
   editing,
 }: Props) => {
-  const formContext = useFormikContext();
+  const formContext = useContext(FormikContext);
```

Comment:
There is a warning in the console when we try to use `useFormikContext()` in non-edit mode. That happens because the `DataTable` component in this case is not inside the Formik provider. Using `useContext(FormikContext);` remove this warnings